packages/docs/src/styles/components/navigation.scss

```diff
@@ -133,42 +138,23 @@
   }
 }
 
+.ds-c-vertical-nav__label {
```

Comment:
moved this rule outside of the desktop media query as the hover state should be available across viewport sizes